stylesheets/stylesheet.css

```diff
@@ -219,7 +219,7 @@ Full-Width Styles
 
 .inner {
   position: relative;
-  max-width: 640px;
+  max-width: 75%;
```

Comment:
@dwalluck I think we should apply this to GME as well. Can you do this please?

Reply:
Sure. I suppose that fixed width makes more sense on smaller devices. You can see it used elsewhere, particularly for other media types. For a large screen, I am not sure why there is such a strict limit.

Reply:
https://github.com/project-ncl/gradle-manipulator/pull/296

Reply:
Thanks!